PKGBUILD

```diff
@@ -0,0 +1,88 @@
+# Maintainer: Jan Alexander Steffens (heftig) <heftig@archlinux.org>
+# Contributor: Jan de Groot <jgc@archlinux.org>
+
+pkgbase=pango
+pkgname=(
+  pango
+  pango-docs
+)
+pkgver=1.54.0
+pkgrel=1
```

Comment:
Please bump this to 2, so it correctly replaces the existing package in `extra` and `cachyos-extra-*`